mms/metrics_manager.py

```diff
@@ -33,21 +33,36 @@ def memory(metric_instance):
     timer.daemon = True
     timer.start()
 
-def disk(metric_instance):
+def disk_percentage(metric_instance):
     disk_usage = psutil.disk_usage('/').percent / 100.0
     metric_instance.update(disk_usage)
 
+    timer = threading.Timer(intervalSec, disk_percentage, [metric_instance])
+    timer.daemon = True
+    timer.start()
+
+def disk(metric_instance):
+    disk_usage = psutil.disk_usage('/').used / (1024 * 1024) # in MB
+    metric_instance.update(disk_usage)
+
     timer = threading.Timer(intervalSec, disk, [metric_instance])
     timer.daemon = True
     timer.start()
 
+Unit = {
+  'ms': "Milliseconds",
+  'percent': 'Percent',
+  'count': 'Count',
+  'mb': 'Megabytes'
```

Comment:
Can we change this to MB, to differentiate from "m" which is milli.

Reply:
Changed.